target/linux/ipq40xx/base-files/lib/preinit/05_set_iface_mac_ipq40xx.sh

```diff
@@ -9,6 +9,11 @@ preinit_set_mac_address() {
 		ip link set dev eth0 address $(macaddr_add "$base_mac" +1)
 		ip link set dev eth1 address $(macaddr_add "$base_mac" +3)
 		;;
+	engenius,eap2200)
+		base_mac=$(cat /sys/class/net/eth0/address)
+		ip link set dev eth0 address $(macaddr_add "$base_mac" +0)
```

Comment:
`$(macaddr_add "$base_mac" +0)` can be replaced with `"${base_mac}"`

Reply:
This can be removed entirely, as this sets eth0 address to eth0 address.

Reply:
Thanks for advice.